### ISSUE TITLE
Kickass address is wrong in config.json

### ISSUE THREAD

**Issue by tubededentifrice** [Stephen304/bitcannon#73]
In default config.json file:

    {
        "name": "KickAssTorrents",
        "url": "https://kickass.so/hourlydump.txt.gz",
        "frequency": 3600
    }

Correct address is: `https://kickass.to/hourlydump.txt.gz`

**Comment by Stephen304** [comment]
Yep I fixed that in my release config, but since I haven't been able to get much done lately, I haven't made a new release with the proper url.

**Comment by OmgImAlexis** [comment]
The correct address now is `https://kat.cr/hourlydump.txt.gz` and you need to apply for API access.

**Comment by XxUnkn0wnxX** [comment]
i have API access but i cannot find the user hash where do i find this?

EDIT: just copied the download link from one of the dumps had my hash in there...
